polyfills/fm/client/fmradio.js

```diff
@@ -0,0 +1,553 @@
+
+  Object.defineProperty(window.navigator.mozFMRadio, 'onenabled', {
+    set: function(cb) {
+      this._onenabled = cb;
+      this._onenabledId = this._onenabledId || ++_currentRequestId;
+      var commandObject = {
+        serialize: function() {
+          return {
+            id: ++_currentRequestId,
+            data: {
+              operation: 'onpropertychange',
+              handler: 'onenabled',
+              property: 'enabled'
+            },
+            processAnswer: answer => {
+              if (answer.result) {
+                window.navigator.mozFMRadio.enabled =
+                  answer.result.propertyValue;
+                cb(answer.result);
+              }
+            }
+          };
+        }
+      };
+      navConnPromise.then(navConnHelper =>
+        navConnHelper.sendObject(commandObject));
+    }
+  });
+
+  Object.defineProperty(window.navigator.mozFMRadio, 'ondisabled', {
+    set: function(cb) {
+      this._ondisabled = cb;
+      this._ondisabledId = this._ondisabledId || ++_currentRequestId;
+      var commandObject = {
+        serialize: function() {
+          return {
+            id: ++_currentRequestId,
+            data: {
+              operation: 'onpropertychange',
+              handler: 'ondisabled',
+              property: 'enabled'
+            },
+            processAnswer: answer => {
+              if (answer.result) {
+                window.navigator.mozFMRadio.enabled =
+                  answer.result.propertyValue;
+                cb(answer.result);
+              }
+            }
+          };
+        }
+      };
+      navConnPromise.then(navConnHelper =>
+        navConnHelper.sendObject(commandObject));
+    }
+  });
+
+  Object.defineProperty(window.navigator.mozFMRadio,
+    'onantennaavailablechange', {
+    set: function(cb) {
+      this._onantennaavailablechange = cb;
+      this._onantennaavailablechangeId = this._onantennaavailablechangeId ||
+        ++_currentRequestId;
+      var commandObject = {
+        serialize: function() {
+          return {
+            id: ++_currentRequestId,
+            data: {
+              operation: 'onpropertychange',
+              handler: 'onantennaavailablechange',
+              property: 'antennaAvailable'
+            },
+            processAnswer: answer => {
+              if (answer.result) {
+                window.navigator.mozFMRadio.antennaAvailable =
+                  answer.result.propertyValue;
+                cb(answer.result);
+              }
+            }
+          };
+        }
+      };
+      navConnPromise.then(navConnHelper =>
+        navConnHelper.sendObject(commandObject));
+    }
+  });
+
+  /** POLYFILL PART **/
+  var navConnPromise = new NavConnectHelper();
+
+  navConnPromise.then(function(){}, e => {
+    debug('Got an exception while connecting ' + e);
+    window.navigator.mozFMRadio = null;
+  });
+
+  ////////////////////////////////////////////////////////////////////////////////
```

Comment:
And as the comment says, we can move this to a common file now ;) But leave that for now, I can do that myself once this is landed.